distribution/docs/src/main/resources/_contents/_data/security-attributes-table-contents.adoc

```diff
@@ -0,0 +1,26 @@
+﻿
+.[[_security_attributes_table]]Security: Attributes in this group relate to security of the resource and metadata.
+[cols="1,2,1,1,1" options="header"]
+|===
+
+|Term
+|Definition
+|Datatype
+|Constraints
+|Example Value
+
+|security.access-groups
+|Attribute name for storing groups to enforce access
+controls upon
+|String
+|
+|
+
+|security.access-individuals
+|Attribute name for storing the email addresses of users to
```

Comment:
Should we include an example to show how to list the email addresses or multi-valued attributes in general? Maybe I'm missing a note that says these should be comma-delimited.